docs/router.md

```diff
@@ -0,0 +1,216 @@
+# About Routers
+
+`gpii.express.router` is a subclass of [`gpii.express.middleware`](middleware.md) that wraps
+[the  Express `router` object](http://expressjs.com/en/4x/api.html#router), which is simply a container for a collection
+of [middleware](middleware.md).
+
+Both Express and the Express `router` object have the concept of a "stack" of middleware.  A `gpii.express.router`
```

Comment:
Within the context of each of these "containers", there is an actual `stack` variable, which is why I chose the name.